LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2018, E3SM:  Energy Exascale Earth System Model
```

Comment:
Is the copyright year for E3SM or for `zstash`? If the latter, should it be the year `zstash` was created or the year we're adding the license?

Reply:
It should be for z-stash.  It should always be the current year or a range from the year that the code was started to the current year.

Reply:
I should also say that you probably can't just add a license.  You may need to consult with LLNL about the details here.